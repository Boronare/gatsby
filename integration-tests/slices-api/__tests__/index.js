/**
 * These tests ensure that Slices API output the correct data files and HTML to the public directory
 */

const { readdirSync, readFileSync } = require(`fs-extra`)
const path = require(`path`)

const publicDir = `${__dirname}/../public`
const slicesDataDir = `${publicDir}/slice-data`
const slicesHTMLDir = `${publicDir}/_gatsby/slices`

describe(`Slices API`, () => {
  describe(`Slices data`, () => {
    const dataFilePaths = readdirSync(path.resolve(slicesDataDir))

    it(`Should create a data file for each slice`, () => {
      expect(dataFilePaths).toMatchSnapshot()
    })

    it(`Files created for each slice should contain the correct data`, () => {
      dataFilePaths.forEach(dataFilePath => {
        const dataFile = readFileSync(`${slicesDataDir}/${dataFilePath}`)
        expect(dataFile).toMatchSnapshot()
      })
    })
  })

  describe(`Slices HTML`, () => {
    const htmlFilePaths = readdirSync(path.resolve(slicesHTMLDir))

    it(`Should create an HTML file for each slice`, () => {
      expect(htmlFilePaths).toMatchSnapshot()
    })

    it(`Files created for each slice should contain the correct HTML`, () => {
      htmlFilePaths.forEach(htmlFilePath => {
        const htmlFile = readFileSync(`${slicesHTMLDir}/${htmlFilePath}`)
        expect(htmlFile).toMatchSnapshot()
      })
    })
  })
})
