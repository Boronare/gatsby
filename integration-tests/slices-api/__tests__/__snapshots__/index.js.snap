// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 1`] = `
Object {
  "data": Array [
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    60,
    115,
    99,
    114,
    105,
    112,
    116,
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    105,
    100,
    61,
    34,
    103,
    97,
    116,
    115,
    98,
    121,
    45,
    99,
    104,
    117,
    110,
    107,
    45,
    109,
    97,
    112,
    112,
    105,
    110,
    103,
    34,
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    62,
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    119,
    105,
    110,
    100,
    111,
    119,
    46,
    95,
    95,
    95,
    99,
    104,
    117,
    110,
    107,
    77,
    97,
    112,
    112,
    105,
    110,
    103,
    61,
    34,
    123,
    92,
    34,
    97,
    112,
    112,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    97,
    112,
    112,
    45,
    51,
    48,
    101,
    49,
    102,
    57,
    57,
    101,
    55,
    49,
    99,
    102,
    51,
    48,
    55,
    53,
    99,
    99,
    53,
    98,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    112,
    97,
    103,
    101,
    115,
    45,
    52,
    48,
    52,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    112,
    97,
    103,
    101,
    115,
    45,
    52,
    48,
    52,
    45,
    106,
    115,
    45,
    55,
    54,
    102,
    99,
    100,
    55,
    56,
    98,
    53,
    54,
    50,
    50,
    101,
    50,
    48,
    101,
    48,
    100,
    56,
    48,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    112,
    97,
    103,
    101,
    115,
    45,
    105,
    110,
    100,
    101,
    120,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    112,
    97,
    103,
    101,
    115,
    45,
    105,
    110,
    100,
    101,
    120,
    45,
    106,
    115,
    45,
    54,
    97,
    52,
    56,
    102,
    98,
    56,
    52,
    57,
    53,
    56,
    56,
    51,
    49,
    98,
    99,
    102,
    52,
    98,
    101,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    116,
    101,
    109,
    112,
    108,
    97,
    116,
    101,
    115,
    45,
    98,
    108,
    111,
    103,
    45,
    112,
    111,
    115,
    116,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    116,
    101,
    109,
    112,
    108,
    97,
    116,
    101,
    115,
    45,
    98,
    108,
    111,
    103,
    45,
    112,
    111,
    115,
    116,
    45,
    106,
    115,
    45,
    100,
    52,
    56,
    56,
    49,
    55,
    51,
    53,
    50,
    57,
    56,
    51,
    99,
    100,
    97,
    56,
    99,
    48,
    57,
    99,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    98,
    105,
    111,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    98,
    105,
    111,
    45,
    106,
    115,
    45,
    53,
    56,
    50,
    49,
    98,
    100,
    98,
    57,
    99,
    54,
    102,
    57,
    57,
    102,
    48,
    51,
    53,
    52,
    101,
    51,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    102,
    111,
    111,
    116,
    101,
    114,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    102,
    111,
    111,
    116,
    101,
    114,
    45,
    106,
    115,
    45,
    50,
    55,
    57,
    52,
    55,
    98,
    97,
    54,
    56,
    50,
    99,
    52,
    53,
    48,
    53,
    51,
    56,
    56,
    50,
    53,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    45,
    106,
    115,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    45,
    106,
    115,
    45,
    57,
    55,
    53,
    48,
    54,
    55,
    102,
    102,
    97,
    56,
    99,
    53,
    102,
    101,
    98,
    98,
    98,
    99,
    55,
    49,
    46,
    106,
    115,
    92,
    34,
    93,
    44,
    92,
    34,
    103,
    97,
    116,
    115,
    98,
    121,
    45,
    112,
    108,
    117,
    103,
    105,
    110,
    45,
    105,
    109,
    97,
    103,
    101,
    45,
    111,
    98,
    106,
    101,
    99,
    116,
    102,
    105,
    116,
    45,
    112,
    111,
    108,
    121,
    102,
    105,
    108,
    108,
    92,
    34,
    58,
    91,
    92,
    34,
    47,
    103,
    97,
    116,
    115,
    98,
    121,
    45,
    112,
    108,
    117,
    103,
    105,
    110,
    45,
    105,
    109,
    97,
    103,
    101,
    45,
    111,
    98,
    106,
    101,
    99,
    116,
    102,
    105,
    116,
    45,
    112,
    111,
    108,
    121,
    102,
    105,
    108,
    108,
    45,
    57,
    54,
    55,
    49,
    51,
    99,
    54,
    53,
    101,
    49,
    53,
    98,
    55,
    53,
    54,
    98,
    48,
    57,
    100,
    53,
    46,
    106,
    115,
    92,
    34,
    93,
    125,
    34,
    59,
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    60,
    47,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
    10,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    32,
    60,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
    119,
    105,
    110,
    100,
    111,
    119,
    46,
    95,
    95,
    95,
    119,
    101,
    98,
    112,
    97,
    99,
    107,
    67,
    111,
    109,
    112,
    105,
    108,
    97,
    116,
    105,
    111,
    110,
    72,
    97,
    115,
    104,
    61,
    34,
    98,
    52,
    52,
    53,
    54,
    100,
    57,
    49,
    57,
    99,
    97,
    101,
    49,
    49,
    98,
    54,
    99,
    50,
    100,
    49,
    34,
    59,
    60,
    47,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
    60,
    115,
    99,
    114,
    105,
    112,
    116,
    32,
    115,
    114,
    99,
    61,
    34,
    47,
    119,
    101,
    98,
    112,
    97,
    99,
    107,
    45,
    114,
    117,
    110,
    116,
    105,
    109,
    101,
    45,
    97,
    101,
    55,
    97,
    54,
    97,
    56,
    50,
    52,
    98,
    97,
    48,
    54,
    52,
    97,
    97,
    55,
    97,
    48,
    53,
    46,
    106,
    115,
    34,
    32,
    97,
    115,
    121,
    110,
    99,
    62,
    60,
    47,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
    60,
    115,
    99,
    114,
    105,
    112,
    116,
    32,
    115,
    114,
    99,
    61,
    34,
    47,
    102,
    114,
    97,
    109,
    101,
    119,
    111,
    114,
    107,
    45,
    56,
    100,
    99,
    49,
    55,
    102,
    101,
    56,
    53,
    50,
    98,
    97,
    56,
    48,
    98,
    98,
    102,
    100,
    50,
    100,
    46,
    106,
    115,
    34,
    32,
    97,
    115,
    121,
    110,
    99,
    62,
    60,
    47,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
    60,
    115,
    99,
    114,
    105,
    112,
    116,
    32,
    115,
    114,
    99,
    61,
    34,
    47,
    97,
    112,
    112,
    45,
    51,
    48,
    101,
    49,
    102,
    57,
    57,
    101,
    55,
    49,
    99,
    102,
    51,
    48,
    55,
    53,
    99,
    99,
    53,
    98,
    46,
    106,
    115,
    34,
    32,
    97,
    115,
    121,
    110,
    99,
    62,
    60,
    47,
    115,
    99,
    114,
    105,
    112,
    116,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 2`] = `
Object {
  "data": Array [
    60,
    100,
    105,
    118,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    98,
    105,
    111,
    34,
    62,
    60,
    112,
    62,
    87,
    114,
    105,
    116,
    116,
    101,
    110,
    32,
    98,
    121,
    32,
    60,
    115,
    116,
    114,
    111,
    110,
    103,
    62,
    74,
    111,
    115,
    104,
    32,
    74,
    111,
    104,
    110,
    115,
    111,
    110,
    60,
    47,
    115,
    116,
    114,
    111,
    110,
    103,
    62,
    32,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    119,
    104,
    111,
    32,
    108,
    105,
    118,
    101,
    115,
    32,
    97,
    110,
    100,
    32,
    119,
    111,
    114,
    107,
    115,
    32,
    105,
    110,
    32,
    77,
    105,
    99,
    104,
    105,
    103,
    97,
    110,
    32,
    98,
    117,
    105,
    108,
    100,
    105,
    110,
    103,
    32,
    110,
    101,
    97,
    116,
    32,
    116,
    104,
    105,
    110,
    103,
    115,
    46,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    32,
    60,
    97,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    104,
    116,
    116,
    112,
    115,
    58,
    47,
    47,
    116,
    119,
    105,
    116,
    116,
    101,
    114,
    46,
    99,
    111,
    109,
    47,
    48,
    120,
    74,
    48,
    53,
    72,
    34,
    62,
    89,
    111,
    117,
    32,
    115,
    104,
    111,
    117,
    108,
    100,
    32,
    102,
    111,
    108,
    108,
    111,
    119,
    32,
    116,
    104,
    101,
    109,
    32,
    111,
    110,
    32,
    84,
    119,
    105,
    116,
    116,
    101,
    114,
    60,
    47,
    97,
    62,
    60,
    47,
    112,
    62,
    60,
    47,
    100,
    105,
    118,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 3`] = `
Object {
  "data": Array [
    60,
    100,
    105,
    118,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    98,
    105,
    111,
    34,
    62,
    60,
    112,
    62,
    87,
    114,
    105,
    116,
    116,
    101,
    110,
    32,
    98,
    121,
    32,
    60,
    115,
    116,
    114,
    111,
    110,
    103,
    62,
    75,
    121,
    108,
    101,
    32,
    77,
    97,
    116,
    104,
    101,
    119,
    115,
    60,
    47,
    115,
    116,
    114,
    111,
    110,
    103,
    62,
    32,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    119,
    104,
    111,
    32,
    108,
    105,
    118,
    101,
    115,
    32,
    97,
    110,
    100,
    32,
    119,
    111,
    114,
    107,
    115,
    32,
    105,
    110,
    32,
    83,
    97,
    110,
    32,
    70,
    114,
    97,
    110,
    99,
    105,
    115,
    99,
    111,
    32,
    98,
    117,
    105,
    108,
    100,
    105,
    110,
    103,
    32,
    117,
    115,
    101,
    102,
    117,
    108,
    32,
    116,
    104,
    105,
    110,
    103,
    115,
    46,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    32,
    60,
    97,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    104,
    116,
    116,
    112,
    115,
    58,
    47,
    47,
    116,
    119,
    105,
    116,
    116,
    101,
    114,
    46,
    99,
    111,
    109,
    47,
    107,
    121,
    108,
    101,
    109,
    97,
    116,
    104,
    101,
    119,
    115,
    34,
    62,
    89,
    111,
    117,
    32,
    115,
    104,
    111,
    117,
    108,
    100,
    32,
    102,
    111,
    108,
    108,
    111,
    119,
    32,
    116,
    104,
    101,
    109,
    32,
    111,
    110,
    32,
    84,
    119,
    105,
    116,
    116,
    101,
    114,
    60,
    47,
    97,
    62,
    60,
    47,
    112,
    62,
    60,
    47,
    100,
    105,
    118,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 4`] = `
Object {
  "data": Array [
    60,
    102,
    111,
    111,
    116,
    101,
    114,
    62,
    194,
    169,
    32,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    50,
    48,
    50,
    50,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    44,
    32,
    66,
    117,
    105,
    108,
    116,
    32,
    119,
    105,
    116,
    104,
    60,
    33,
    45,
    45,
    32,
    45,
    45,
    62,
    32,
    60,
    97,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    104,
    116,
    116,
    112,
    115,
    58,
    47,
    47,
    119,
    119,
    119,
    46,
    103,
    97,
    116,
    115,
    98,
    121,
    106,
    115,
    46,
    99,
    111,
    109,
    34,
    62,
    71,
    97,
    116,
    115,
    98,
    121,
    60,
    47,
    97,
    62,
    32,
    60,
    105,
    62,
    60,
    97,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    104,
    116,
    116,
    112,
    115,
    58,
    47,
    47,
    119,
    119,
    119,
    46,
    103,
    97,
    116,
    115,
    98,
    121,
    106,
    115,
    46,
    99,
    111,
    109,
    47,
    100,
    111,
    99,
    115,
    47,
    114,
    101,
    102,
    101,
    114,
    101,
    110,
    99,
    101,
    47,
    98,
    117,
    105,
    108,
    116,
    45,
    105,
    110,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    47,
    103,
    97,
    116,
    115,
    98,
    121,
    45,
    115,
    108,
    105,
    99,
    101,
    34,
    62,
    40,
    97,
    110,
    100,
    32,
    83,
    108,
    105,
    99,
    101,
    115,
    33,
    41,
    60,
    47,
    97,
    62,
    60,
    47,
    105,
    62,
    60,
    47,
    102,
    111,
    111,
    116,
    101,
    114,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 5`] = `
Object {
  "data": Array [
    60,
    104,
    101,
    97,
    100,
    101,
    114,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    103,
    108,
    111,
    98,
    97,
    108,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    34,
    62,
    60,
    104,
    49,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    109,
    97,
    105,
    110,
    45,
    104,
    101,
    97,
    100,
    105,
    110,
    103,
    34,
    62,
    60,
    97,
    32,
    97,
    114,
    105,
    97,
    45,
    99,
    117,
    114,
    114,
    101,
    110,
    116,
    61,
    34,
    112,
    97,
    103,
    101,
    34,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    34,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    47,
    34,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 6`] = `
Object {
  "data": Array [
    60,
    47,
    97,
    62,
    60,
    47,
    104,
    49,
    62,
    60,
    47,
    104,
    101,
    97,
    100,
    101,
    114,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 7`] = `
Object {
  "data": Array [
    60,
    104,
    101,
    97,
    100,
    101,
    114,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    103,
    108,
    111,
    98,
    97,
    108,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    34,
    62,
    60,
    97,
    32,
    97,
    114,
    105,
    97,
    45,
    99,
    117,
    114,
    114,
    101,
    110,
    116,
    61,
    34,
    112,
    97,
    103,
    101,
    34,
    32,
    99,
    108,
    97,
    115,
    115,
    61,
    34,
    104,
    101,
    97,
    100,
    101,
    114,
    45,
    108,
    105,
    110,
    107,
    45,
    104,
    111,
    109,
    101,
    34,
    32,
    104,
    114,
    101,
    102,
    61,
    34,
    47,
    34,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Files created for each slice should contain the correct HTML 8`] = `
Object {
  "data": Array [
    60,
    47,
    97,
    62,
    60,
    47,
    104,
    101,
    97,
    100,
    101,
    114,
    62,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices HTML Should create an HTML file for each slice 1`] = `
Array [
  "_gatsby-scripts-1.html",
  "bio--joshj-1.html",
  "bio--kylem-1.html",
  "footer-1.html",
  "header-3e94a830accf80530978d746d9728ec0-1.html",
  "header-3e94a830accf80530978d746d9728ec0-2.html",
  "header-ad79b1a24008dc2beea0a23ad9cbff97-1.html",
  "header-ad79b1a24008dc2beea0a23ad9cbff97-2.html",
]
`;

exports[`Slices API Slices data Files created for each slice should contain the correct data 1`] = `
Object {
  "data": Array [
    123,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    67,
    104,
    117,
    110,
    107,
    78,
    97,
    109,
    101,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    98,
    105,
    111,
    45,
    106,
    115,
    34,
    44,
    34,
    114,
    101,
    115,
    117,
    108,
    116,
    34,
    58,
    123,
    34,
    100,
    97,
    116,
    97,
    34,
    58,
    123,
    34,
    97,
    117,
    116,
    104,
    111,
    114,
    34,
    58,
    123,
    34,
    110,
    97,
    109,
    101,
    34,
    58,
    34,
    74,
    111,
    115,
    104,
    32,
    74,
    111,
    104,
    110,
    115,
    111,
    110,
    34,
    44,
    34,
    115,
    117,
    109,
    109,
    97,
    114,
    121,
    34,
    58,
    34,
    119,
    104,
    111,
    32,
    108,
    105,
    118,
    101,
    115,
    32,
    97,
    110,
    100,
    32,
    119,
    111,
    114,
    107,
    115,
    32,
    105,
    110,
    32,
    77,
    105,
    99,
    104,
    105,
    103,
    97,
    110,
    32,
    98,
    117,
    105,
    108,
    100,
    105,
    110,
    103,
    32,
    110,
    101,
    97,
    116,
    32,
    116,
    104,
    105,
    110,
    103,
    115,
    46,
    34,
    44,
    34,
    116,
    119,
    105,
    116,
    116,
    101,
    114,
    34,
    58,
    34,
    48,
    120,
    74,
    48,
    53,
    72,
    34,
    125,
    125,
    44,
    34,
    115,
    108,
    105,
    99,
    101,
    67,
    111,
    110,
    116,
    101,
    120,
    116,
    34,
    58,
    123,
    34,
    112,
    97,
    116,
    104,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    98,
    105,
    111,
    45,
    45,
    106,
    111,
    115,
    104,
    106,
    34,
    44,
    34,
    105,
    100,
    34,
    58,
    34,
    106,
    111,
    115,
    104,
    106,
    34,
    125,
    125,
    44,
    34,
    115,
    116,
    97,
    116,
    105,
    99,
    81,
    117,
    101,
    114,
    121,
    72,
    97,
    115,
    104,
    101,
    115,
    34,
    58,
    91,
    93,
    125,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices data Files created for each slice should contain the correct data 2`] = `
Object {
  "data": Array [
    123,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    67,
    104,
    117,
    110,
    107,
    78,
    97,
    109,
    101,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    98,
    105,
    111,
    45,
    106,
    115,
    34,
    44,
    34,
    114,
    101,
    115,
    117,
    108,
    116,
    34,
    58,
    123,
    34,
    100,
    97,
    116,
    97,
    34,
    58,
    123,
    34,
    97,
    117,
    116,
    104,
    111,
    114,
    34,
    58,
    123,
    34,
    110,
    97,
    109,
    101,
    34,
    58,
    34,
    75,
    121,
    108,
    101,
    32,
    77,
    97,
    116,
    104,
    101,
    119,
    115,
    34,
    44,
    34,
    115,
    117,
    109,
    109,
    97,
    114,
    121,
    34,
    58,
    34,
    119,
    104,
    111,
    32,
    108,
    105,
    118,
    101,
    115,
    32,
    97,
    110,
    100,
    32,
    119,
    111,
    114,
    107,
    115,
    32,
    105,
    110,
    32,
    83,
    97,
    110,
    32,
    70,
    114,
    97,
    110,
    99,
    105,
    115,
    99,
    111,
    32,
    98,
    117,
    105,
    108,
    100,
    105,
    110,
    103,
    32,
    117,
    115,
    101,
    102,
    117,
    108,
    32,
    116,
    104,
    105,
    110,
    103,
    115,
    46,
    34,
    44,
    34,
    116,
    119,
    105,
    116,
    116,
    101,
    114,
    34,
    58,
    34,
    107,
    121,
    108,
    101,
    109,
    97,
    116,
    104,
    101,
    119,
    115,
    34,
    125,
    125,
    44,
    34,
    115,
    108,
    105,
    99,
    101,
    67,
    111,
    110,
    116,
    101,
    120,
    116,
    34,
    58,
    123,
    34,
    112,
    97,
    116,
    104,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    98,
    105,
    111,
    45,
    45,
    107,
    121,
    108,
    101,
    109,
    34,
    44,
    34,
    105,
    100,
    34,
    58,
    34,
    107,
    121,
    108,
    101,
    109,
    34,
    125,
    125,
    44,
    34,
    115,
    116,
    97,
    116,
    105,
    99,
    81,
    117,
    101,
    114,
    121,
    72,
    97,
    115,
    104,
    101,
    115,
    34,
    58,
    91,
    93,
    125,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices data Files created for each slice should contain the correct data 3`] = `
Object {
  "data": Array [
    123,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    67,
    104,
    117,
    110,
    107,
    78,
    97,
    109,
    101,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    102,
    111,
    111,
    116,
    101,
    114,
    45,
    106,
    115,
    34,
    44,
    34,
    114,
    101,
    115,
    117,
    108,
    116,
    34,
    58,
    123,
    34,
    115,
    108,
    105,
    99,
    101,
    67,
    111,
    110,
    116,
    101,
    120,
    116,
    34,
    58,
    123,
    34,
    112,
    97,
    116,
    104,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    102,
    111,
    111,
    116,
    101,
    114,
    34,
    125,
    125,
    44,
    34,
    115,
    116,
    97,
    116,
    105,
    99,
    81,
    117,
    101,
    114,
    121,
    72,
    97,
    115,
    104,
    101,
    115,
    34,
    58,
    91,
    93,
    125,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices data Files created for each slice should contain the correct data 4`] = `
Object {
  "data": Array [
    123,
    34,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    67,
    104,
    117,
    110,
    107,
    78,
    97,
    109,
    101,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    45,
    115,
    114,
    99,
    45,
    99,
    111,
    109,
    112,
    111,
    110,
    101,
    110,
    116,
    115,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    45,
    106,
    115,
    34,
    44,
    34,
    114,
    101,
    115,
    117,
    108,
    116,
    34,
    58,
    123,
    34,
    115,
    108,
    105,
    99,
    101,
    67,
    111,
    110,
    116,
    101,
    120,
    116,
    34,
    58,
    123,
    34,
    112,
    97,
    116,
    104,
    34,
    58,
    34,
    115,
    108,
    105,
    99,
    101,
    45,
    45,
    104,
    101,
    97,
    100,
    101,
    114,
    34,
    125,
    125,
    44,
    34,
    115,
    116,
    97,
    116,
    105,
    99,
    81,
    117,
    101,
    114,
    121,
    72,
    97,
    115,
    104,
    101,
    115,
    34,
    58,
    91,
    93,
    125,
  ],
  "type": "Buffer",
}
`;

exports[`Slices API Slices data Should create a data file for each slice 1`] = `
Array [
  "bio--joshj.json",
  "bio--kylem.json",
  "footer.json",
  "header.json",
]
`;
